[2024-01-19T03:46:48.108+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T03:46:47.575353+00:00 [queued]>
[2024-01-19T03:46:48.113+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T03:46:47.575353+00:00 [queued]>
[2024-01-19T03:46:48.114+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-19T03:46:48.120+0000] {taskinstance.py:2192} INFO - Executing <Task(BranchPythonOperator): CheckTableExisting> on 2024-01-19 03:46:47.575353+00:00
[2024-01-19T03:46:48.126+0000] {standard_task_runner.py:60} INFO - Started process 54 to run task
[2024-01-19T03:46:48.132+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'company_profile_pipeline', 'CheckTableExisting', 'manual__2024-01-19T03:46:47.575353+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/CrawlCompanyInfo.py', '--cfg-path', '/tmp/tmpfgnfnqek']
[2024-01-19T03:46:48.133+0000] {standard_task_runner.py:88} INFO - Job 192: Subtask CheckTableExisting
[2024-01-19T03:46:48.166+0000] {task_command.py:423} INFO - Running <TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T03:46:47.575353+00:00 [running]> on host a56b2d2a6d76
[2024-01-19T03:46:48.204+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='anh.ho@tititada.com' AIRFLOW_CTX_DAG_OWNER='anh.ho' AIRFLOW_CTX_DAG_ID='company_profile_pipeline' AIRFLOW_CTX_TASK_ID='CheckTableExisting' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T03:46:47.575353+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-19T03:46:47.575353+00:00'
[2024-01-19T03:46:48.205+0000] {logging_mixin.py:188} INFO - Connecting to the PostgreSQL database...
[2024-01-19T03:46:48.210+0000] {logging_mixin.py:188} INFO - PostgreSQL database version:
[2024-01-19T03:46:48.211+0000] {logging_mixin.py:188} INFO - Successfully connect to Postgresql
[2024-01-19T03:46:48.211+0000] {logging_mixin.py:188} INFO - ('PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit',)
[2024-01-19T03:46:48.212+0000] {logging_mixin.py:188} INFO - Checking existing table...
[2024-01-19T03:46:48.215+0000] {python.py:201} INFO - Done. Returned value was: CreateCompanyProfileTable
[2024-01-19T03:46:48.215+0000] {branch.py:35} INFO - Branch into CreateCompanyProfileTable
[2024-01-19T03:46:48.215+0000] {skipmixin.py:178} INFO - Following branch CreateCompanyProfileTable
[2024-01-19T03:46:48.225+0000] {skipmixin.py:244} INFO - Skipping tasks [('ExistingCompanyProfile', -1)]
[2024-01-19T03:46:48.251+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=company_profile_pipeline, task_id=CheckTableExisting, execution_date=20240119T034647, start_date=20240119T034648, end_date=20240119T034648
[2024-01-19T03:46:48.272+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-19T03:46:48.287+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
